android/app/src/google-services.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
Can you move this file to path "android/app/google-services.json" or add it to .gitignore? So it won't be a part of the open-source repo. Users of the repo can add their own google-services.json file.